test/persistent_receiver_test.go

```diff
@@ -148,6 +150,32 @@ var _ = Describe("PersistentReceiver", func() {
 			Expect(err).ToNot(HaveOccurred())
 			Expect(fmt.Sprint(receiver)).To(ContainSubstring(fmt.Sprintf("%p", receiver)))
 		})
+		Context("with a connected messaging service and specially formatted queue", func() {
+            slog.SetLogLoggerLevel(slog.LevelDebug)
```

Comment:
@TrentDaniel, please any reason why you included this ?

Reply:
I included that for debugging purposes, I will turn it off now, good catch